test/test_netmhc_cons.py

```diff
@@ -0,0 +1,20 @@
+from mhctools import NetMHCcons
+from mhctools.alleles import normalize_allele_name
+
+
+DEFAULT_ALLELE = 'HLA-A*02:01'
+
+def test_netmhc_pan():
```

Comment:
Why is the file named `test_netmhc_cons.py` and the test named `test_netmhc_pan`?